<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditVersion">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Device </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PlayBack Device </string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxRecordingDevice">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPlayoutDevice">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>361</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderMicrophoneVolume">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRecording">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPlayout">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Volume</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PlayBack Volume</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderSpeakerVolume">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMicrophoneVolumeValue">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSpeakerVolumeValue">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Voice Engine</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Sound Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Play Sound Level</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarRecord">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarPlay">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
